User.API/Validators/ShortClientViewModelValidator.cs

```diff
@@ -18,8 +18,10 @@ public ShortClientViewModelValidator()
         RuleFor(c => c.Email)
             .EmailAddress();
 
-        RuleFor(c => c.PhoneNumber)
-            .Matches(@"^(\+\d{1,3}\s?)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$")
-            .When(c => c.PhoneNumber is not null).WithMessage("Phone number must match pattern\nExample: 1234567890 +9211234567890");
+        RuleFor(c => c.Password)
+            .MinimumLength(8)
+            .MaximumLength(20)
```

Comment:
So, if the password is longer than 20 symbols then the message will be _"Weak Password"_